README.md

```diff
@@ -85,43 +102,56 @@ tar xzf models.tgz
 
 ## Usage
 
+You need to obtain a pre-trained model that capture naming
+conventions.  The default pre-trained model, which was trained using
+our [corpus][math-comp-corpus] and follows the conventions used in the
+[Mathematical Components][math-comp-website] family of projects, can
+be obtained by running the command:
+
+```
+./bin/roosterize download_global_model
+```
+
+The model will be downloaded to `$HOME/.roosterize/`. To use a
+different model (that we [released][latest-release] or you trained),
+simply put it in `$HOME/.roosterize/`.
```

Comment:
This part should go before the "Usage" subsection, under "Installation of Roosterize and trained models" just above, replacing the current content there.